Zend/zend_vm_def.h

```diff
@@ -9808,6 +9808,44 @@ ZEND_VM_HANDLER(209, ZEND_INIT_PARENT_PROPERTY_HOOK_CALL, CONST, UNUSED|NUM, NUM
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED, UNUSED)
```

Comment:
```suggestion
ZEND_VM_HANDLER(210, ZEND_FETCH_DEFAULT_ARG, UNUSED|NUM, UNUSED)
```

Reply:
Considering this appears to work despite this, why is this needed?

Reply:
1. Documentation 2. It's used when dumping the oparray.